libs/application/template-api-modules/src/lib/modules/templates/new-primary-school/new-primary-school.service.ts

```diff
@@ -38,13 +38,96 @@ export class NewPrimarySchoolService extends BaseTemplateApiService {
   }
 
   async getChildren({ auth }: TemplateApiModuleActionProps) {
-    const children =
+    let children =
       await this.nationalRegistryService.getChildrenCustodyInformation(auth)
 
     const currentYear = new Date().getFullYear()
     const maxYear = currentYear - 7 // 2nd grade
     const minYear = currentYear - 16 // 10th grade
 
+    // Mock test children
+    if (isRunningOnEnvironment('local') || isRunningOnEnvironment('dev')) {
+      if (auth.audkenniSimNumber == '0103019') {
+        children = [
+          {
+            nationalId: '2007104360',
```

Comment:
Do the same for lines 59, 65, 71, 77, 86, 92 and 98

`Found valid SSN: 2203122590 eslint(local-rules/disallow-kennitalas)`

```suggestion
            // eslint-disable-next-line local-rules/disallow-kennitalas
            nationalId: '2007104360',
```

Reply:
Added disabled eslint checker for kennitala for testing purposes